src/views/Swap.vue

```diff
@@ -432,6 +440,7 @@ export default class Swap extends Mixins(TranslationMixin, LoadingMixin, NumberF
       }
       await this.updatePairLiquiditySources()
       await this.recountSwapValues()
+      await this.checkLiquidity()
```

Comment:
recountSwapValues is a long running async function, i think the flow should be
```
await Promise.all([this.updatePairLiquiditySources(), this.checkLiquidity()])
await this.recountSwapValues()
```
Because `updatePairLiquiditySources ` `checkLiquidity` working with independent data, but `recountSwapValues` uses data from `updatePairLiquiditySources `

Also, i think we should add a call of `this.checkLiquidity` to the `created `hook, if both tokens are selected